bin/sl-nginx-ctl.js

```diff
@@ -24,15 +23,19 @@ var parser = new Parser([
     ':v(version)',
     'h(help)',
     'b:(base)',
-    'c:(control)',
-    'l:(listen)',
+    'A:(api)',
+    'n(no-api)',
+    'C:(control)',
+    'R:(routable-addr)',
```

Comment:
would be nice if `-A` and `-R` meant the same thing in our various products, if that is reasonably possible